### ISSUE TITLE
How to save the model?

### ISSUE THREAD

**Issue by gerald-dotcom** [rudikershaw/whichx#6]
Any way to do this?

**Comment by rudikershaw** [comment]
If you have a small dataset (x < 100,000) I would advise storing the data for your model separately in your preferred format and then rebuilding this model each time. Doing it this way will prevent you from tightly coupling your code to this library. 

However, this is isn't always practical. If this is impractical in your case then you can do the following;
```js
var whichpet = new WhichX();
// ... Build your model here.
var model = whichpet.export();
```

That model can then be stored and used to pre-populate another WhichX object.

```js
var whichpet = new WhichX();
var model = retrieveModel(); 
whichpet.export(model);
```

I hope this helps.

**Comment by gerald-dotcom** [comment]
Alright, I'm having an issue with a multi class classification problem, your lib always output "pet". Why is that?

**Comment by rudikershaw** [comment]
That generally occurs when all your words are registered against a single label. The 'pet' is a default that comes from the example implementation. 

This has been fixed in 2.1.3 as far as I am aware.

**Comment by gerald-dotcom** [comment]
it works amazing. I thought I would need a Multinomial Naive Bayesian classifier.
